components/camel-jq/src/generated/resources/META-INF/services/org/apache/camel/other.properties

```diff
@@ -0,0 +1,7 @@
+# Generated by camel build tools - do NOT edit this file!
```

Comment:
I think the other.properties is wrong, maybe try to `rm -rf src/generated` to regen from fresh